cmd/agent/gui/views/templates/generalStatus.tmpl

```diff
@@ -388,6 +388,11 @@
         {{- if .SketchesFlushErrors}}
           Sketches Flush Errors: {{.SketchesFlushErrors}}<br>
         {{- end -}}
+        {{- if .EventPlatformEvents }}
+        {{- range $k, $v := .EventPlatformEvents }}
+          {{ $k }}: {{humanize $v}}
+        {{- end }}
+        {{- end }}
```

Comment:
I don't think having a subsection is going to be useful because the grouping of "event platform events" isn't a meaningful user-facing concept. See the rest of my comments here https://github.com/DataDog/datadog-agent/pull/7735#discussion_r613258957